### ISSUE TITLE
Attempt to load IPv6 when requesting over IPv4

### ISSUE THREAD

**Issue by Wouter0100** [wtfismyip/wtfismyip#10]
Just like when the request is IPv6 it'll load IPv4 using ipv4.wtfismyip.com, it would come in handy if it's the same the other way around: IPv4 load IPv6.

I've unfortunately had survival cases when Chrome fail back to IPv4 due to an aggressive Happy Eyeballs implementation.

**Comment by wtfismyip** [comment]
Hi Wouter,

Sorry for the late reply!

The current strategy for serving HTML content on https://wtfismyip.com/ is to provide both an A and AAAA record in DNS. If the request is made over IPv6, then JavaScript is loaded to display the IPv4 address through ipv4.wtfismyip.com/js

There may be a better strategy to handle the case you described. Will think about it.

**Comment by Wouter0100** [comment]
Yeah, I saw how that worked. That seems like the way to go, but it would be awesome if it's possibly to add the same for IPv6: if the page is loaded using ipv4, try to load IPv6 using JS.

**Comment by wtfismyip** [comment]
Perhaps we could load all content through JavaScript - if enabled - both IPv4 and IPv6 addresses, and have a fallback if JS isn't enabled.

**Comment by Wouter0100** [comment]
That seems like a neat solution. Fallback would just be the current way of rendering the IP in a <noscript>, right?

**Comment by antreos** [comment]
My 2 cents: 
For some reasons my browser prefers IPv4 over IPv6. I only see my IPv4 address, but can't force to visit the IPv6 Check as there is none like for IPv4. So something like ipv4.wtfismyip.com for IPv6 would be neat.

**Comment by Wouter0100** [comment]
@AndiKnight you know that https://ipv6.wtfismyip.com exists? 😜

**Comment by antreos** [comment]
@Wouter0100 Well, now I do. Thanks.
Could swear I tried it some time ago and it didn't worked - my fault sorry for bugging you.
